packages/api/src/controllers/webhooksController.ts

```diff
@@ -115,6 +126,59 @@ export default async function webhookController(fastify: FastifyInstance) {
     },
   );
 
+  fastify.withTypeProvider<TypeBoxTypeProvider>().post(
+    "/amazonses",
+    {
+      schema: {
+        description: "Used to consume amazonses notification events.",
+        tags: ["Webhooks"],
+        body: AmazonSNSEvent,
+      },
+      onRequest: (req, _, done) => {
+        // eslint-disable-next-line no-param-reassign
+        req.headers["content-type"] = "application/json";
+        done();
+      },
+    },
+    async (request, reply) => {
+      logger().debug({ body: request.body }, "Received AmazonSES event.");
+
+      // Validate the signature
+      const valid = await validSNSSignature(request.body);
+
+      if (valid.isErr()) {
+        logger().error("Invalid signature for AmazonSES webhook.", valid.error);
+        return reply.status(401).send({ message: "Invalid signature" });
+      }
+
+      switch (request.body.Type) {
+        case AmazonSNSEventTypes.SubscriptionConfirmation:
```

Comment:
The effect of this code is to have the `SubscriptionConfirmation` and `UnsubscribeConfirmation` be handled by the same code block. Is that the intention?

Reply:
Yes, indeed.  I can refactor if you'd prefer not use a fall through case. (Its also a three case switch.  It would be just as clean as if/else.  Your call ofc.

Reply:
This is fine, just write a short comment please! Philosophically opposed to bikeshedding code style, as long as the intent is clear.